.github/workflows/helm-release.yaml

```diff
@@ -72,29 +77,68 @@ jobs:
         with:
           ref: ${{ inputs.ref }}
 
-      - name: Set up python
-        uses: actions/setup-python@v2
-        with:
-          python-version: "${{ inputs.python-version }}"
-
       - name: Bump version
         id: release-tag
-        uses: bakdata/ci-templates/actions/bump-version@v1.0.0
+        uses: bakdata/ci-templates/actions/bump-version@v1.5.2
         with:
           release-type: ${{ inputs.release-type }}
+          working-directory: ${{ inputs.charts-dir }}
+
+      - name: Download previous Helm index & packages
+        run: curl -O ${{ inputs.page-url }}/index.yaml
```

Comment:
Did you check what happens if this doesn't exist?

Reply:
Okay, after a quick local check: This would create a file with the following content
```html
<html>
<head><title>301 Moved Permanently</title></head>
<body>
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

and therefore would lead to an error in the next step, because it's not JSON.

Reply:
good point, I'll change it to skip if the file doesn't exist yet.

Reply:
Went for a more robust approach: If curl doesn't find a file, the step fails. For the initial setup of the workflow there is a `skip-download` input option, because there are no files yet on GH pages.